scripts/core/editor3/html/to-html/plainTextToHtml.ts

```diff
@@ -0,0 +1,6 @@
+// converts linebreaks in paragraphs
+export const plainTextToHtml = (text: string) =>
+    text.replace(/.*[^\n]/g, '<p>$&</p>');
+
+export const stringIsHtml = (text: string) =>
```

Comment:
I'd use `DOMParser` for this https://stackoverflow.com/a/15458968/1175593

I'd also name it `isStringHtml`. Adding `is` in front makes it more of a question which reads better for functions while `stringIsHtml` is more of a statement which is better for booleans: `const stringIsHtml = isStringHtml(str)`.